vim_menutrans_helper.py

```diff
@@ -0,0 +1,228 @@
+    # non-english character
+    with open(untranslation_file, encoding="latin1") as f1:
+        for line_number, line in enumerate(f1):
+            to_be_translated_words = []
+
+            line = line.strip()
+            temp_word_list = []
+            pre_end = -1
+            top_string = line.split()
+            if len(top_string) > 0 and top_string[0] in exe_commands:
+                # Extract content inside quotes from line
+                for match in re.finditer(
+                        r"'(?:''|[^'])*'|(?:\"(?:(?:\\.)|[^\"])*\")", line):
+                    mid_part = line[pre_end + 1: match.start() - 1].strip()
+
+                    # Add a digit, namely "5", if there are variables inside 
+                    # mid_part
+                    if not ((len(mid_part) == 1 and mid_part[0] == ".") or 
+                            len(mid_part) == 0 or pre_end == -1):
+                        temp_word_list.append("5")
+                    pre_end = match.end()
+                    temp_word_list.append(line[match.start() + 1: 
+                            match.end() - 1])
+
+                temp_line = ""
+                for word in temp_word_list:
+                    if len(word) == 0:
+                        continue
+                    if word[0] == "'":
+                        word = unescape_single_quotes(word)
+                    else:
+                        word = unescape_double_quotes(word)
+                    temp_line += word
+                line = temp_line
+
+            word_list = []
+            # Split line string according to blank without backslash or tab 
+            # ahead
+            for match in re.finditer(r"(?:(?:\\.)|[^ \t])+", line):
+                word_list.append(line[match.start(): match.end()])
+
+            if len(word_list) == 0 or word_list[0] == '"':
+                continue
+            index = 0
+            # Menu_item with "disable" or enable followed close behind should 
+            # be ignored
+            if (word_list[0] in menu_commands and len(word_list) > 1 and 
+                    (word_list[1] == "disable" or word_list[1] == "enable")):
+                continue
+
+            if word_list[0] in sil_commands:
+                index += 1
+            # Blocks needs to be translated without being split by '.'
+            if index < len(word_list) and word_list[index] in menu_commands:
+                to_be_translated_blocks = ""
+                index += 1
+                while index < len(word_list):
+                    if (word_list[index] == '<silent>' or 
+                            word_list[index] == '<script>' or 
+                            (word_list[index][0].isdigit() and 
+                            not word_list[index - 1][0].isdigit())):
+                        index += 1
+                    else:
+                        # Menu_item with "ToolBar" has no following words to be 
+                        # translated
```

Comment:
The comment is not accurate. It's actually `A menu item under "ToolBar" is not an end-user facing string, but just an identity to be references by ":tmenu" command, and will be translated by another ":tmenu" command. Therefore the menu item itself doesn't require a translation.`